src/fiatExchanges/quotes/ExternalQuote.ts

```diff
@@ -76,10 +79,10 @@ export default class ExternalQuote extends NormalizedQuote {
     return isSimplexQuote(this.quote) ? this.provider.paymentMethods[0] : this.quote.paymentMethod
   }
 
-  getCryptoType(): CiCoCurrency {
+  getCryptoType(): string {
     return isSimplexQuote(this.quote)
-      ? resolveCICOCurrency(this.quote.digital_money.currency)!
-      : resolveCICOCurrency(this.quote.digitalAsset)!
+      ? this.quote.digital_money.currency!
+      : this.quote.digitalAsset!
```

Comment:
```suggestion
      ? this.quote.digital_money.currency
      : this.quote.digitalAsset
```
these aren't needed anymore. hooray!